config/utils.go

```diff
@@ -180,3 +181,52 @@ func JSONToLogTargetCfg(data []byte) (mlog.LogTargetCfg, error) {
 	}
 	return cfg, nil
 }
+
+func GetConfigValueByPath(config *model.Config, path []string) (interface{}, bool) {
```

Comment:
Why this wrapper function here?

Reply:
@jespino - we didn't have a function for getting config values by dotted path, so I've copied one from mmctl

Reply:
Yes, still, I don't think the wrapper is providing any value.